utils/managers.py

```diff
@@ -624,9 +630,6 @@ def filter_cubes(self, unfilteredCubes, filters, mode):
                     else:
                         addFlag = False
                         break
-                """ Below two lines are added temorarily. TODO: Fix download of seasonal data """
```

Comment:
We should filter out the seasonal data that does not work at the moment.

Reply:
Solved.